tools/probes/common/check_rse_attributes

```diff
@@ -111,10 +113,14 @@ if __name__ == '__main__':
             add_rse_attribute(name, 'spacetoken', str(rse['token']), session=session)
             add_rse_attribute(name, 'site', str(rse['site']), session=session)
             add_rse_attribute(name, 'type', str(rse['type']), session=session)
-            if str(rse['space_method']) == 'other':
+            if rse['space_usage_url']:
                 add_rse_attribute(name, 'space_usage_method', 'json', session=session)
             elif name in SPACE_USAGE:
                 del_rse_attribute(name, 'space_usage_method', session=session)
+            if rse['space_method'] == 'rucio':
+                add_rse_attribute(name, 'sourceForUsedSpace', 'rucio', session=session)
```

Comment:
this is the first time i see CamelCase instead of snake_case for RSE attributes. any reason to break the convention?

Reply:
This was introduced two years ago with commit 863225cf42b3bc9bb5a140a58bcc50706708ad94.